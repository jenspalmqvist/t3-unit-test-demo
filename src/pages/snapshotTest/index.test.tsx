import React from "react";
import renderer from "react-test-renderer";
import SnapshotTest from ".";

it("should render correctly", () => {
  const tree = renderer.create(<SnapshotTest />).toJSON();
  expect(tree).toMatchSnapshot();
});
