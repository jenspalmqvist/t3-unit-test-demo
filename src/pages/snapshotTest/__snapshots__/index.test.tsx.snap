// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render correctly 1`] = `
<div>
  Snapshot test 2
</div>
`;
